packages/notifi-react-card/lib/assets/BackArrow copy.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
I think that this file does not need to be committed - I think that you are using the original back arrow file?

Reply:
as the back arrow icons are not exactly the same(same, color, weight), I renamed the back arrow as IntercomBackArrow.

Reply:
We could pass in a different color and size but yes, I know that could be hard.